src/tabletop.js

```diff
@@ -101,6 +101,11 @@
       this.log("You passed a new Google Spreadsheets url as the key! Attempting to parse.");
       this.key = this.key.match("d\\/(.*?)\\/pubhtml")[1];
     }
+    
+    if(/spreadsheets\/d/.test(this.key)) {
+      this.log("You passed a new Google Spreadsheets url as the key! Attempting to parse.");
```

Comment:
You could change this log or the one above(line 101) to indicate the different version of google spreadsheet URL that is being parsed.